MentoringHomeWork/HW6/HW6.Task1/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using HW6.Task1;
+
+var arrayCalculateSum = new ArrayCalculateSum();
+ArrayCalculateSum.ArraySum();
```

Comment:
You used the static method of the ArrayCalculateSum.ArraySum();
It's not necessary to initialize a new variable and object for this above - you don't use it actually but you used memory.

Reply:
resolved